about.html

```diff
@@ -0,0 +1,106 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/fontawesome.min.css" integrity="sha512-SgaqKKxJDQ/tAUAAXzvxZz33rmn7leYDYfBP+YoMRSENhf3zJyx3SBASt/OfeQwBHA1nxMis7mM3EV/oYT6Fdw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="about.css">
+    <title>HennaHubuk</title>
+</head>
+<body>
+    <body>
+        <header>
+          <i class="fa-solid fa-bars" aria-hidden="true"></i>
+          <div class="top-header">
+              <ul>
+                  <li><a href="#"><i class="fa-brands fa-facebook-f"></i></a></li>
+                  <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                  <li><a href="#">English</a></li>
+                  <li><a href="#">My Page</a></li>
+                  <li><a href="#">Logout</a></li>
+              </ul>
+            </div>
+            <div class="sub-header">
+                <a href="index.html"><img src="./images/logo.jfif" alt="website_logo"></a>
+          <div class="header-right">
+            <nav>
+              <ul class="links">
+                <li><a class="nav-link active" href="index.html">Home</a></li>
+                <li><a class="nav-link" href="#">Program</a></li>
+                <li><a class="nav-link" href="#">Join</a></li>
+                <li><a class="nav-link" href="#">Sponsor</a></li>
+                <i class="fa-solid fa-xmark"></i>
+              </ul>
+              <Span>Special Seat</Span>
+            </nav>
+            </div>
+        </header>
+        <div class="begin-about">
+                <div class="intro-about">
+                    <p>"Hello HennaHub Globe!"</p>
+                    <h1 class="intro-trans-anim">HENNAHUBUK'S CREATIVE DESIGN APPRENTICE SUMMIT <br>2023</h1>
+                </div>
+                <div>
+                    <p class="about-text">
+                       @ HennaHub, we offer henna tattoo of any type for any occasion with over 20 years experience, glitter tattoo and face painting. We also offer henna designs using body paint, patch test of 24 to 48 hours is mandatory before full henna application incase of allergies and sensitive skin to 100% plant product. The henna types we use are: 
+                       <br> Natural red henna and Natural black jagua henna 
+                    </p>
+                </div>
+                <div class="contact-us">
+                    Please Contact us per Email, for further enquiry about HennaHubuk apprentice summit 2023 <br>
+                    <span>hennahubuk@gmail.com</span>
+                  </div>
+                </div>
```

Comment:
- [ ] Same here as pointed earlier :point_up_2: Kindly wrap this with a `section` tag.